npm-shrinkwrap.json

```diff
@@ -8742,6 +8742,7 @@
       "version": "7.5.0",
       "resolved": "https://registry.npmjs.org/sinon/-/sinon-7.5.0.tgz",
       "integrity": "sha512-AoD0oJWerp0/rY9czP/D6hDTTUYGpObhZjMpd7Cl/A6+j0xBE+ayL/ldfggkBXUs0IkvIiM1ljM8+WkOc5k78Q==",
+      "deprecated": "16.1.1",
```

Comment:
Not sure why this happened but it happened

Reply:
I wasn't able to get that change from the `logic-function` branch. Can you remove this change and try to do a clean install? 👉  `npm ci`

Don't forget to use `nvm use`